public/app/features/dashboard-scene/sharing/ExportButton/ExportAsImage.tsx

```diff
@@ -0,0 +1,294 @@
+      <div data-testid={selectors.components.ExportImage.preview.error.title}>{error.title}</div>
+      <div data-testid={selectors.components.ExportImage.preview.error.message}>{error.message}</div>
+    </Alert>
+  );
+}
+
+function ImagePreview({ imageBlob, isLoading }: { imageBlob: Blob | null; isLoading: boolean }) {
+  const styles = useStyles2(getStyles);
+
+  if (!imageBlob || isLoading) {
+    return null;
+  }
+
+  return (
+    <img
+      src={URL.createObjectURL(imageBlob)}
+      alt={t('share-modal.image.preview', 'Preview')}
+      className={styles.image}
+      data-testid={selectors.components.ExportImage.preview.image}
+      aria-label={t('share-modal.image.preview-aria', 'Generated dashboard image preview')}
+      role="img"
+    />
+  );
+}
+
+function RendererAlert() {
+  if (config.rendererAvailable) {
+    return null;
+  }
+
+  return (
+    <Alert
+      severity="info"
+      title={t('share-modal.link.render-alert', 'Image renderer plugin not installed')}
+      data-testid={selectors.components.ExportImage.rendererAlert.container}
+    >
+      <div data-testid={selectors.components.ExportImage.rendererAlert.title}>
+        {t('share-modal.link.render-alert', 'Image renderer plugin not installed')}
+      </div>
+      <div data-testid={selectors.components.ExportImage.rendererAlert.description}>
+        <Trans i18nKey="share-modal.link.render-instructions">
+          To render a panel image, you must install the{' '}
+          <a
+            href="https://grafana.com/grafana/plugins/grafana-image-renderer"
+            target="_blank"
+            rel="noopener noreferrer"
+            className="external-link"
+          >
+            Grafana image renderer plugin
+          </a>
+          . Please contact your Grafana administrator to install the plugin.
+        </Trans>
+      </div>
+    </Alert>
+  );
+}
+
+function ExportAsImageRenderer({ model }: SceneComponentProps<ExportAsImage>) {
+  const [format, setFormat] = useState<ImageFormat>(ImageFormat.PNG);
+  const [isLoading, setIsLoading] = useState(false);
+  const [imageBlob, setImageBlob] = useState<Blob | null>(null);
+  const [error, setError] = useState<ErrorState>(null);
+  const styles = useStyles2(getStyles);
+  const [ref, { width: loadingBarWidth }] = useMeasure<HTMLDivElement>();
+
+  const dashboard = getDashboardSceneFor(model);
+
+  // Clean up object URLs when component unmounts
+  useEffect(() => {
+    return () => {
+      if (imageBlob) {
+        URL.revokeObjectURL(URL.createObjectURL(imageBlob));
+      }
+    };
+  }, [imageBlob]);
+
+  const onFormatChange = (value: ImageFormat) => {
+    setFormat(value);
+  };
+
+  const onExport = async () => {
+    setIsLoading(true);
+    setError(null);
+
+    try {
+      const result = await generateDashboardImage({
+        dashboard,
+        format,
+        scale: config.rendererDefaultImageScale || 1,
```

Comment:
Maybe this could be an option in the UI (as we currently have in the panel image sharing workflow).